test/fixtures/linter/autofix/ansi/008_looping_rules_l003_l016_l019/after.sql

```diff
@@ -2,7 +2,7 @@ SELECT
     COUNT(1) AS campaign_count,
     state_user_v_peer_open,
     business_type,
-    
+
```

Comment:
Incidental change to output. The new behavior seems better (completely blank line rather than line with 4 spaces).